arangodb-net-standard/AuthApi/AuthApiClient.cs

```diff
@@ -0,0 +1,40 @@
+﻿using ArangoDBNetStandard.Serialization;
+using ArangoDBNetStandard.Transport;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ArangoDBNetStandard.AuthApi
+{
+    public class AuthApiClient: ApiClientBase
+    {
+        private IApiClientTransport _client;
+
+        public AuthApiClient(IApiClientTransport client)
+            : base(new JsonNetApiClientSerialization())
+        {
+            _client = client;
+        }
+
+        public AuthApiClient(IApiClientTransport client, IApiClientSerialization serializer)
+            : base(serializer)
+        {
+            _client = client;
+        }
+
+        public async Task<JwtTokenResponse> GetJwtToken(JwtTokenRequestBody body)
```

Comment:
I wonder about the naming here. The request is a `POST` but the method is prefixed with `Get`, which sounds appropriate, but that's not in line with most of the other endpoint names.

Reply:
I reasoned that since it's a POST request used as a command, rather than a POST request used to create a new resource, that it would be OK.  Similar to how we have `TruncateCollectionAsync` instead of `PutCollectionTruncateAsync` or something.

Reply:
Although I just noticed it should have `async` suffix I think.